source/includes/_overview.md

```diff
@@ -0,0 +1,11 @@
+# Introduction
+
+You can integrate Tipser in your website in multiple ways. The fastest way is to use [Tipser Elements](#tipser-elements) or [Tipser React Elements](#tipser-react-elements) - a complete toolkit of native commerce experience that requires less than 5 minutes of tech setup.
```

Comment:
I certainly agree about Tipser Elements. But I'm not sure if we should advertise Tipser React Element as "5 minutes solution" 🤔 . Tipser Elements React is more code-heavy then Tipser Elements (can't be done just via simple modifications in the CMS), so it may require a full release cycle. Also understanding which components to use - there are (too) many components publicly exported and they are not well documented - and ensuring a proper component hierarchy may take some time to grasp.